backend/src/main/java/aiversum/backend/rest/UserConfigController.java

```diff
@@ -0,0 +1,28 @@
+package aiversum.backend.rest;
+
+import aiversum.backend.model.UserConfig;
+import aiversum.backend.service.UserConfigService;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api/config")
+public class UserConfigController {
+
+    private final UserConfigService userConfigService;
+
+    public UserConfigController(UserConfigService userConfigService) {
+        this.userConfigService = userConfigService;
+    }
+
+    @GetMapping("/{email}")
+    public UserConfig getUserConfig(@PathVariable String email){
+        return userConfigService.getConfigByEmail(email);
```

Comment:
With this configuration each user can retrieve other user's config. You should retrieve the user data from SecurityContext